template/page-uses.html.jinja

```diff
@@ -345,86 +358,153 @@
    <section class="container-new py-26 text-lg-center">
     <div class="p-3 col-lg-8 mx-lg-auto">
       <div class="d-flex flex-column-reverse">
-        <h1 class="mb-0">{% trans %}Proven Powerful for Innovation{% endtrans %}</h1>
-        <h6 class="eyebrow mb-3">{% trans %}XRPL Use Cases{% endtrans %}</h6>
+        <h1 class="mb-0">{% trans %}Powering Innovative Use Cases and Projects.{% endtrans %}</h1>
+        <h6 class="eyebrow mb-3">{% trans %}XRPL Ecosystem{% endtrans %}</h6>
       </div>
     </div>
   </section>
 
   <section class="container-new py-26">
-    <div class="col-lg-6 p-3 mb-3 mb-lg-5">
+    <div class="col-lg-5 p-3">
       <div class="d-flex flex-column-reverse">
-        <h3 class="h4 h2-sm">{% trans %}Building businesses and creating new value{% endtrans %}</h3>
-        <h6 class="eyebrow mb-3">{% trans %}XRPL Ecosystem{% endtrans %}</h6>
+      <div class="d-flex justify-content-start align-items-center"> 
+        <div class="arrow-animation" id="arrowAnimation"> </div>
+        <span class="explore-projects">Explore Featured Projects </span>
+      </div>
+        <p class="text-sm">{% trans %}The XRPL has a rich ecosystem with many contributors globally. Explore the community of developers, validators, and partners.{% endtrans %}</p>
+        <h6 class="eyebrow mb-3">{% trans %}Introducing the XRPL Ecosystem{% endtrans %}</h6>
       </div>
     </div>
+    <div class="col-lg-5 offset-lg-2 p-5 d-flex">
+          <div class="mb-4 pb-3 numbers-animation" id="numbersAnimation"></div>
+          <div class="mb-4 pb-3 numbers-animation" id="numbersAnimationLight"></div>
+          <div class="apps-built">Apps/exchanges <br/> built on the <br/> XRPL </div>
+    </div>
+
 
-    <ul class="card-grid card-grid-3xN ls-none mt-4 pt-lg-2" id="use-case-card-grid">
+    <ul class="card-grid card-grid-4xN ls-none mt-4 pt-lg-2" id="use-case-card-grid">
       {% set uses = [
-        
           {"id": "infrastructure",
             "title": _("Infrastructure"),
-            "description": _("Build and operate components or systems that help the functionality of the XRP Ledger, such as Nodes, dev tools, storage, security and more.")},
+            "number": 7,
+            "description" : "Build and operate components or systems that help the functionality of the XRP Ledger, such as Nodes, dev tools, storage, security and more."
```

Comment:
The `_(...)` is there to make the strings translatable. If you remove it like this, the Japanese translation won't display properly.

Reply:
Was not aware, added them back.